### ISSUE TITLE
Write more examples

### ISSUE THREAD

**Issue by CoffeeStraw** [CoffeeStraw/PyonFX#9]
Examples are never enough. Any contributions showing particular usage of implemented functions would be appreciated.

**Comment by Youka** [comment]
After some refactoring of available kfx scripts with more comments and cleaner code they could serve as good examples. Reviewing them and collecting different levels of difficulty would result in a helpful set.

The only (old) problem would be copyright - many people just copy effects and publish as their own work so contibutors should be wary of that!

**Comment by CoffeeStraw** [comment]
Good point, probably I need to update contribution guidelines and add a readme in the examples directory.
